apps/expo-go/ios/Exponent/Kernel/AppLoader/EXHomeStubLoader.m

```diff
@@ -0,0 +1,91 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#import "EXHomeStubLoader.h"
+#import "EXVersions.h"
+#import "EXBuildConstants.h"
+
+#import <React/RCTUtils.h>
+
+@import EXManifests;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EXHomeStubLoader ()
```

Comment:
In the future, it will be nice to actually remove the stub and make a separate path that handles home, as we don't need to load it anymore.